### PR TITLE
:sparkles: Add filter for urgency

### PR DESCRIPTION
**Pull request by tcmetzger**
Proposed fix for https://trello.com/c/lYAC9dXS to differentiate between "Vorabinformation" und "Warnung" by filtering by urgency.

### CODE REVIEW

**Comment by jh0ker** [comment]
Could you also use this filter in `db.query` to apply this filter to the feed? You can simply add another commit to this git branch.

**Comment by tcmetzger** [comment]
Filter to db query-function (based on the model of how we filter for states) submitted in additional commit.

**Comment by jh0ker** [comment]
That's not a good way to implement this filter. The reason why the states filter is so complicated is that an event has a list of states associated with it. Filtering by certainty should be modeled after the severity filter (like you did for the slack/ftp logic), since an event always has only one certainty, like it has only one severity.

File: unwetter/db.py [hunk 1]
```diff
@@ -97,6 +98,10 @@ def query(severities, states, limit=50):
     else:
         filter['$or'] = [{'states': state} for state in states]
 
+    filter = {
```
Comment (jh0ker) [inline unwetter/db.py]: This overwrites the `filter` variable.

File: unwetter/db.py [hunk 2]
```diff
@@ -97,6 +98,10 @@ def query(severities, states, limit=50):
...
+        'urgency': {'$in': urgencies},
```
Comment (jh0ker) [inline unwetter/db.py]: This line should be inserted after line 93 to be combined with the other filters

**Comment by tcmetzger** [comment]
New commit to append filter instead of replacing it.

**Pull request merged**